core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/IntelliJPaths.scala

```diff
@@ -1,41 +1,43 @@
 package org.virtuslab.ideprobe.ide.intellij
 
+import org.virtuslab.ideprobe.Config
+
 import java.nio.file.Path
 import org.virtuslab.ideprobe.Extensions._
 
-final class IntelliJPaths(root: Path, headless: Boolean) {
-  val config: Path = root.createDirectory("config")
-  val system: Path = root.createDirectory("system")
-  val plugins: Path = root.createDirectory("plugins")
-  val logs: Path = system.createDirectory("logs")
+final case class IntelliJPaths(
+    root: Path,
+    config: Path,
+    system: Path,
+    plugins: Path,
+    logs: Path,
+    userPrefs: Path
+) {
   val bin: Path = root.resolve("bin")
-  val userPrefs: Path = {
-    val path = root.resolve("prefs")
-    IntellijPrivacyPolicy.installAgreementIn(path)
-    path
-  }
-
-  val executable: Path = {
-    val content = {
-      val launcher = bin.resolve("idea.sh").makeExecutable()
+}
 
-      val command =
-        if (headless) s"$launcher headless"
-        else {
-          Display.Mode match {
-            case Display.Native => s"$launcher"
-            case Display.Xvfb   => s"xvfb-run --server-num=${Display.XvfbDisplayId} $launcher"
-          }
-        }
+object IntelliJPaths {
+  def fromFile(root: Path): IntelliJPaths = {
+    val bin = root.resolve("bin")
+    val ideaProperties = Config.fromFile(bin.resolve("idea.properties"))
 
-      s"""|#!/bin/sh
-          |$command "$$@"
-          |""".stripMargin
+    val configPath = ideaProperties.get[Path]("idea.config.path").getOrElse(root.createDirectory("config"))
+    val systemPath = ideaProperties.get[Path]("idea.system.path").getOrElse(root.createDirectory("system"))
+    val pluginsPath = ideaProperties.get[Path]("idea.plugins.path").getOrElse(root.createDirectory("plugins"))
+    val logsPath = ideaProperties.get[Path]("idea.log.path").getOrElse(root.createDirectory("logs"))
+    val userPrefsPath = ideaProperties.get[Path]("java.util.prefs.userRoot").getOrElse {
```

Comment:
@MichalBednarz is it safe to assume that existing ij will have the privacy policy agreement set up? Or should we set it up for both variants?

Reply:
Setting privacy policy agreement is being taken care of in the case that I'm going to use this feature for. We can skip it for existing instance for now.

Reply:
Done - no longer trying to install the policy.